### PR TITLE
React Capston Web App. Readme file  updated for final review

### PR DESCRIPTION
**Pull request by codeWithHyder**
This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.

You will select coingeco's API that provides numeric data about the topic that you like and then build the webapp around it. The webapp will have several pages:

- one page with a list of items that could be filtered by some parameters; in the example below, it's a list of metrics that can be filtered by the country (imagine a search field to introduce the country name like Italy, Croatia, etc.). This page should be your homepage.
- one page for the item details, the detail page for selected card  description.

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @codeWithHyder, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that uses the old readme file format, kindly use the latest which can be found [here](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by codeWithHyder** [comment]
ah

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- App works perfectly ✔️
- Used redux ✔️


 ## Required Changes ♻️

- [ ] **_🛑 as part of the requirements, the presentation video should contain a highlight about a piece of the code the student finds interesting or is proud of making. kindly retake your video and mention a piece of code you are proud of making_**


- [ ] **_🛑  as part of the requirements, you should follow the design guideline including the Layout: (composition and space between elements. ) , kindly fix it 🙏_**


Your Design :
image 

Guideline:
image

- [x] **_🛑 as part of the requirements, you should test some of your functions including redux and not only your components, kindly add that 🙏 ._**






## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 1]
```diff
@@ -141,27 +139,34 @@ a:hover {
...
-  padding: 1rem;
-  background-color: rgb(46 47 53 / 80.1%);
+  padding-left: 0.5rem;
+  background-color: rgba(27, 28, 29, 0.801);
   backdrop-filter: blur(120px);
-  border-radius: 1rem;
+  border-radius: 0.3rem;
+  border: 0.1rem solid goldenrod;
   transition: ease-in-out 0.5s;
+  width: 100%;
```
Comment (tufoinnkuo10) [inline src/App.css:164]: - [ ] you have a failing test, please look into that. Also, as mentioned by the previous reviewer, kindly test some of your functions including Redux and not only your components,
Screenshot (994)

File: src/App.css [hunk 2]
```diff
@@ -141,27 +139,34 @@ a:hover {
...
 
 .coin-card {
-  gap: 1rem;
-  justify-content: center;
+  column-gap: 0.02rem;
+  justify-content: left;
   align-items: center;
-  padding: 1rem;
-  background-color: rgb(46 47 53 / 80.1%);
+  padding-left: 0.5rem;
```
Comment (tufoinnkuo10) [inline src/App.css]: - [ ] great job making it grid. please like the previous reviwer noted, kindly try to follow the design guideline including the Layout: (composition and space between elements. ). please add the sizes of the cards and make their margins/borders thin and light like in the original design. 

yours              |  original
:-------------------------:|:-------------------------:
Screenshot (995)_LI  |  Screenshot (996)


- [optional] please, you can add the arrows on the cards and the thick and light colors, so it matches well with the design.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @codeWithHyder 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ design followed

video presentation
- please always present your self clearly and the purpose of your project. also, try as much as possible for your face to be visible throughout the presentation. this is friendly. nice job

Required Changes ♻️
Check the comments under the review
Optional suggestions
- [optional] Please always follow the Gitflow when a project requires you to (dev branch as a base branch,). you can you different branch to work and then open a pull request into the dev. this will help the reviewer see your changes. also don't merge when your project is yet to be approved. 😄 


tufoin

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/App.css (hunk 2 above)
Comment (codeWithHyder) [reply src/App.css:162]: If I were given one minute to check someone's PR then there is always room for improvements  i have seen worse in design and logic  projects got approved than this one ,,,,.but i am very much preplexed about the rules you follow while checking PRs of trainees

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @codeWithHyder👋, 

### You did a marvelous job on the React capstone project and this is one of the best projects I have seen so far!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image


## To Highlight 🎉
- All tests are passed. ✔️
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Nice UI design. ✔️
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by codeWithHyder** [comment]
thank you younger brother you deserve all stars

**Pull request merged**